src/createElement.js

```diff
@@ -0,0 +1,36 @@
+export function createElement(type, props, ...children) {
+  props = props || {};
```

Comment:
```suggestion
export function createElement(type, config, ...children) {
  const props = {};
```
처음에는 `props`, `children`을 따로 받고 나중에 `props` 객체에 `children` 속성을 추가해서 합치는 것 같은데 실제 React 코드에서는 매개변수 이름을 `config`로 받고 `const props = {}` 이런 식으로 `props` 객체를 따로 만들더라구요

물론 지금 방식도 문제는 없지만 React는 매개변수 `props` 객체의 형태가 바뀌다보니깐 헷갈릴 수 있어서 구분해놓은 것 같아서 한번 참고해보셔도 좋을 것 같아요~
<a href="https://github.com/facebook/react/blob/main/packages/react/src/jsx/ReactJSXElement.js#L641">createElement 함수 - React 내부 소스 코드</a>